debug.log

```diff
@@ -0,0 +1 @@
+[1010/214642.650:ERROR:settings.cc(430)] Settings version is not 5
```

Comment:
Please add `*.log` to `.gitignore` file and not commit log files which may have elsewhere local changes.